### ISSUE TITLE
Students can easily link to wrong identifier, can't unlink

### ISSUE THREAD

**Issue by amueller** [education/classroom#2542]
### Feature request :sparkles:
From my students experience, it looks like it's very easy to link an account to the wrong identifier. I assume a single click is enough. It would be better if this would require also clicking "ok" or similar.
I had several students link to the wrong UNI.
Once they link to the wrong identifier, they can't unlink.
I had students link to random identifiers that they accidentally clicked on, and now I do not know how to find them in the roster and unlink them.

**Comment by amueller** [comment]
If you could help me unlink that would be great, the github account that doesn't know which identifier is linked to is ilica. I can't search the roster for that from what I can see, so it would be great if you could do that.

**Comment by d12** [comment]
@amueller This is an interesting edge case we hadn't thought of, thanks for letting us know! We'll see what we can do here.

The identifier your student matched to was `ipm2111`.

**Comment by amueller** [comment]
This edgecase happened about 5 times in my class of 225 students ;)

**Comment by amueller** [comment]
It's happening more and more times, and together with the students not being able to figure out which identifier they are linked to, it's pretty tricky to fix, as I can's search for their github user accounts.

**Comment by stevecassidy** [comment]
I also see this often. I raised it last year in #2455.  Once a student does this it means that the real owner of the identifier can't select it, compounding the problem. I then have to use the awful roster management interface to try to detangle the associations.

**Comment by amueller** [comment]
Indeed, and I think it's different from #377 which #2455 has been rolled into.

Generally I refrain from calling any OSS project awful unless I wrote it myself, though ;)

Have you experience #2550 by the way? Not sure how big your class is.

**Comment by d12** [comment]
cc https://github.com/education/classroom/issues/2519#issuecomment-592734122

Posting here as well for folks discovering this issue, we've shipped a small improvement to this interface, it now asks students to verify they clicked the correct roster entry before continuing. Thank you both for your feedback, we really appreciate it. We're working on a new system some time soon!
